modules/microvm/virtualization/microvm/microvm-host.nix

```diff
@@ -3,25 +3,30 @@
 {
   config,
   lib,
+  pkgs,
   ...
 }: let
   cfg = config.ghaf.virtualization.microvm-host;
 in
   with lib; {
     options.ghaf.virtualization.microvm-host = {
       enable = mkEnableOption "MicroVM Host";
-      hostNetworkSupport = mkEnableOption "Network support services to run host applications.";
+      networkSupport = mkEnableOption "Network support services to run host applications.";
     };
 
     config = mkIf cfg.enable {
       microvm.host.enable = true;
       ghaf.systemd = {
-        enable = true;
         withName = "host-systemd";
+        enable = true;
+        boot.enable = true;
         withPolkit = true;
-        withTimesyncd = cfg.hostNetworkSupport;
-        withNss = cfg.hostNetworkSupport;
-        withResolved = cfg.hostNetworkSupport;
+        withTpm2Tss = pkgs.stdenv.hostPlatform.isx86;
+        withRepart = true;
```

Comment:
Can you please add `withFido2 = true;` which is needed while running `systemd-cryptenroll` otherwise will get `FIDO2 token support is not installed` error.